HerPublicWebsite/Program.cs

```diff
@@ -20,6 +22,10 @@ public static void Main(string[] args)
             
             var startup = new Startup(builder.Configuration, builder.Environment);
             startup.ConfigureServices(builder.Services);
+            
+            const string culture = "en-GB";
+            Thread.CurrentThread.CurrentCulture = CultureInfo.CreateSpecificCulture(culture);
+            Thread.CurrentThread.CurrentUICulture = CultureInfo.CreateSpecificCulture(culture);

```

Comment:
seems an odd way to do this (threading?) [docs](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.currentculture?view=netcore-3.1#examples) seem to mention something like `CultureInfo.CurrentCulture = new CultureInfo("en-GB", false);` to change culture

Reply:
Yeah, I think I was looking at a slightly weirder example in the docs. 
I've updated this now, I think your suggestion is better. I've tested this new way and it also works.